### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Password reset notifications retired per MC219002

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5744

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -56,6 +56,9 @@ Follow the steps below if you want to set user passwords to expire after a speci
 5. Type how often passwords should expire. Choose a number of days from 14 to 730.
   
 6. In the second box type when users are notified that their password will expire, and then select **Save**. Choose a number of days from 1 to 30.
+
+> [!NOTE]
+> Password Expiration Notifications are no longer supported in Office Portal or any Office apps except outlook.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
> Password expiration notifications are no longer supported in the Office 365 portal or any Office apps except Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**